stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -441,6 +443,48 @@ extension ExecutorJob {
   }
 }
 
+// Helper to create a trampoline job to execute a job on a specified
+// executor.
+extension ExecutorJob {
+
+  /// Create a trampoline to enqueue the specified job on the specified
+  /// executor.
+  ///
+  /// This is useful in conjunction with the `Clock.run()` API, which
```

Comment:
I think this'll link well: 
```suggestion
  /// This is useful in conjunction with the ``Clock.run(_:at:tolerance:)`` API, which
```